src/sentry/web/urls.py

```diff
@@ -399,6 +399,12 @@ def init_all_applications():
     url(r'^(?P<organization_slug>[\w_-]+)/$',
         react_page_view, name='sentry-organization-home'),
     url(r'^organizations/new/$', generic_react_page_view),
+
+    url(
+        r'^organizations/(?P<organization_slug>[\w_-]+)/issues/(?P<issue_id>\d+)/$',
```

Comment:
For my own curiosity/learning, why is this needed?

Reply:
I am honestly not sure why this path had to be explicitly added and wasn't resolving to a react page by default. I couldn't find any other routes that seemed to match this path in this file